python/GafferUI/ColorChooser.py

```diff
@@ -513,6 +631,11 @@ def __init__( self, color=imath.Color3f( 1 ), **kw ) :
 								)
 							)
 
+						spacerHeight = 4
+						self.__spacers["a"] = GafferUI.Spacer( imath.V2i( 0, spacerHeight ), parenting = { "index" : ( 0, 3 ) } )
```

Comment:
There wasn't any user feedback either way on where alpha should go, just that it would be helpful to have a small spacer between triplet groups. I'm separating alpha from RGB since you can now toggle the RGB group on and off, but not alpha (we decided alpha is it's own concept, and allowing it to be toggled would raise confusion with the `Color3f` vs `Color4f` plugs).

I don't have a real strong opinion on it - I could be convinced if you think it's better being grouped with RGB.

Reply:
It might just be the spacer height producing a lot of empty space around the A that is throwing me. At present, the spacing between slider groups is greater than the spacing we maintain between plugs in the Node Editor, which might be a bit much? Dropping `spacerHeight` down to 2 could be a happy medium.

Reply:
In 2dde05b96b34791b51da863c248924c14ffaba8e, I made it 2px spacing, give that a look and see if it feels better.